src/biogeophys/WaterStateBulkType.F90

```diff
@@ -185,8 +197,9 @@ subroutine InitBulkCold(this, bounds, &
     do c = bounds%begc,bounds%endc
        this%int_snow_col(c)           = h2osno_input_col(c) 
        this%snow_persistence_col(c)   = 0._r8
+       if (use_fan) this%h2osoi_tend_tsl_col(c) = 0._r8
```

Comment:
Although I haven't looked closely, my guess is that this line isn't needed: Typically we only need cold start initialization for (a) state variables that persist from one time step to another, (b) variables whose values are needed earlier in the driver loop than when they are set, or (c) variables for which the grid cell average diagnostics depend on having values set to some particular value over (for example) special landunits. (a) and (b) imply that the variable is needed on the restart file, which doesn't seem to be the case here; and the use of l2g_scale_type='veg' for the diagnostic field means that (c) doesn't look like the case here.

It's not a big deal to have this here unnecessarily, but it can help for long-term maintenance if we only have things in initCold that truly need to be there.

Reply:
I think this comment still stands.

Reply:
(This isn't important enough to hold up the PR, but I'm just going back through my comments and making notes of which ones are resolved and which still stand.)